shared/templates/accounts_password/oval.template

```diff
@@ -1,13 +1,15 @@
+{{% if product == "ol8" %}}
+{{% set filepath_regex="^/etc/security/pwquality.conf(\.d/[^/]+\.conf)?$" %}}
+{{% else %}}
+{{% set filepath_regex="^/etc/security/pwquality.conf$" %}}
```

Comment:
The dots before conf should be escaped because a dot can match any character not only literal dot.